scripts/src/java/org/oppia/android/scripts/AccessibilityLabelCheck.kt

```diff
@@ -0,0 +1,112 @@
+package org.oppia.android.scripts
+
+import org.w3c.dom.Node
+import org.w3c.dom.NodeList
+import java.io.File
+import java.util.stream.Collectors
+import java.util.stream.IntStream
+import javax.xml.parsers.DocumentBuilderFactory
+
+/**
+ * Script for ensuring that all the Activities the
+ * repo are defined with accessibility labels.
+ */
+class AccessibilityLabelCheck {
+  companion object {
+    @JvmStatic
+    fun main(vararg args: String) {
+      // path of the repo to be analyzed.
+      val repoPath = args[0] + "/"
+
+      // path of the manifest file relative to the repo.
+      val manifesFilePath = args[1]
+
+      // full path of the manifest file.
+      val fullPathToManifestFile = repoPath + manifesFilePath
+
+      // class object which is needed to acess the helper methods.
+      val accessibilityLabelCheck: AccessibilityLabelCheck = AccessibilityLabelCheck()
+
+      // builder factory which provides the builder to parse the XMl.
+      val builderFactory = DocumentBuilderFactory.newInstance()
+
+      // document builder which parses the XMl.
+      val docBuilder = builderFactory.newDocumentBuilder()
+
+      // parsed DOM of manifest file.
+      val doc = docBuilder.parse(File(fullPathToManifestFile))
+      doc.getDocumentElement().normalize()
+
+      // list of all the activity elements.
+      val nodeList = accessibilityLabelCheck.convertNodeListToListOfNode(
+        doc.getElementsByTagName("activity")
+      )
+
+      // list of activity elements which lacks the label.
+      val matchedNodes = nodeList.filter { node ->
```

Comment:
Name this val as `activityListWithoutLabel`

Reply:
Done.